Stripe/legos/stripe_get_all_refunds/stripe_get_all_refunds.py

```diff
@@ -11,23 +11,19 @@ class InputSchema(BaseModel):
     pass
 
 
-pp = pprint.PrettyPrinter(indent=2)
+def stripe_get_all_refunds_printer(output):
+    if isinstance(output, (list, tuple)):
+        pprint.pprint(output)
+    elif isinstance(output, dict):
+        pprint.pprint(output)
+    else:
+        pprint.pprint(output)
 
 
-def legoPrinter(func):
-    def Printer(*args, **kwargs):
-        output = func(*args, **kwargs)
-        print('\n\n')
-        pp.pprint(output)
-        return output
-    return Printer
-
-
-@legoPrinter
 def stripe_get_all_refunds(handle) -> Dict:
     """stripe_get_all_customers Returns a list of refunds that was perviously created. The
         charges are returned in sorted order, with the most recent charges appearing first.
-        
+
```

Comment:
Please check the comment it says get_all_customers. perviously should be previously. 
This lego is not taking care of pagination? We should ask for max_results as input. Default value being 25. User can put it 0 to get all the results. The lego should take care of pagination.